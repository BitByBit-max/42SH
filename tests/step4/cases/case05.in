#!/bin/bash

string="Hello World"

case "$string" in
    [A-Z]*)
        echo "The string starts with an uppercase letter."
        ;;
    *[0-9]*)
        echo "The string contains a number."
        ;;
    *[!A-Za-z0-9]*)
        echo "The string contains a special character."
        ;;
    *)
        echo "The string does not match any of the above patterns."
        ;;
esac
