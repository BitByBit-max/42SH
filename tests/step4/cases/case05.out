The string starts with an uppercase letter.
